package.json

```diff
@@ -75,9 +75,10 @@
     "postbuild": "npm-run-all --parallel copy:*",
     "postinstall": "node postinstall.js",
     "start": "node build/server.js",
-    "test": "npm run test:server",
+    "test": "npm run test:server; npm run test:client",
```

Comment:
@m4dz version recursively calls `"test:build": "env USE_JS=true npm run test"`, do I delete this command, rename it or use @nono's solution ?